tests: benchmarks: build fail in arm/core/fault.c for frdm_k64f in footprint/min

Unrelated fault "fault.c:274:9: error: ‘mmfar’ undeclared" when testing PR #6762:

```
cd /home/buildslave/src/github.com/zephyrproject-rtos/zephyr/out-2nd-pass/frdm_k64f/tests/benchmarks/footprint/benchmark.footprint.min/zephyr && ccache /opt/sdk/zephyr-sdk-0.9.3/sysroots/x86_64-pokysdk-linux/usr/bin/arm-zephyr-eabi/arm-zephyr-eabi-gcc -DBUILD_VERSION=v1.12.0-rc1-225-g0ae6a7cda -DCPU_MK64FN1M0VLL12 -DKERNEL -D_FORTIFY_SOURCE=2 -D__ZEPHYR__=1 -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/kernel/include -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/arch/arm/include -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/arch/arm/soc/nxp_kinetis/k6x -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/arch/arm/soc/nxp_kinetis/k6x/include -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/arch/arm/soc/nxp_kinetis/include -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/boards/arm/frdm_k64f -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/include -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/include/drivers -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/out-2nd-pass/frdm_k64f/tests/benchmarks/footprint/benchmark.footprint.min/zephyr/include/generated -isystem /opt/sdk/zephyr-sdk-0.9.3/sysroots/x86_64-pokysdk-linux/usr/lib/arm-zephyr-eabi/gcc/arm-zephyr-eabi/6.2.0/include -isystem /opt/sdk/zephyr-sdk-0.9.3/sysroots/x86_64-pokysdk-linux/usr/lib/arm-zephyr-eabi/gcc/arm-zephyr-eabi/6.2.0/include-fixed -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/lib/libc/minimal/include -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/ext/hal/cmsis/Include -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/ext/hal/nxp/mcux/devices/MK64F12 -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/ext/hal/nxp/mcux/drivers/kinetis/. -I/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/tests/include  -Os -g -Wall -Wformat -Wformat-security -Wno-format-zero-length -imacros /home/buildslave/src/github.com/zephyrproject-rtos/zephyr/out-2nd-pass/frdm_k64f/tests/benchmarks/footprint/benchmark.footprint.min/zephyr/include/generated/autoconf.h -ffreestanding -Wno-main --sysroot /opt/sdk/zephyr-sdk-0.9.3/sysroots/armv5-zephyr-eabi/usr -mthumb -mcpu=cortex-m4 -std=c99 -Werror -Wno-deprecated-declarations -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-strict-overflow -Wno-pointer-sign -Wno-unused-but-set-variable -fno-reorder-functions -fno-defer-pop -Werror=implicit-int -ffunction-sections -fdata-sections -mabi=aapcs -march=armv7e-m -o CMakeFiles/zephyr.dir/drivers/timer/cortex_m_systick.c.obj   -c /home/buildslave/src/github.com/zephyrproject-rtos/zephyr/drivers/timer/cortex_m_systick.c
/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/arch/arm/core/fault.c: In function ‘_MpuFault’:
/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/arch/arm/core/fault.c:274:9: error: ‘mmfar’ undeclared (first use in this function)
     if (mmfar >= guard_start &&
         ^~~~~
/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/arch/arm/core/fault.c:274:9: note: each undeclared identifier is reported only once for each function it appears in
zephyr/arch/arm/core/CMakeFiles/arch__arm__core.dir/build.make:142: recipe for target 'zephyr/arch/arm/core/CMakeFiles/arch__arm__core.dir/fault.c.obj' failed
make[3]: *** [zephyr/arch/arm/core/CMakeFiles/arch__arm__core.dir/fault.c.obj] Error 1
```

See https://app.shippable.com/github/zephyrproject-rtos/zephyr/runs/15960/2/tests

--- a/arch/arm/core/fault.c
+++ b/arch/arm/core/fault.c
@@ -271,8 +271,8 @@ static u32_t _MpuFault(const NANO_ESF *esf, int fromHardFault)
 #else
 				guard_start = thread->stack_info.start;
 #endif
-				if (mmfar >= guard_start &&
-					mmfar < guard_start +
+				if (SCB->MMFAR >= guard_start &&
+					SCB->MMFAR < guard_start +
 					MPU_GUARD_ALIGN_AND_SIZE) {
 					/* Thread stack corruption */
 					reason = _NANO_ERR_STACK_CHK_FAIL;

